actions/admin/routes.go

```diff
@@ -43,6 +43,7 @@ func RegisterRoutes(handlersManager *handlers.Manager) {
 	// tx
 	adminGroup.GET("/transactions/:id", handlers.AsAdmin(adminGetTxByID))
 	adminGroup.GET("/transactions", handlers.AsAdmin(adminSearchTxs))
+	adminGroup.GET("/transactions/search", handlers.AsAdmin(adminSearchByXPubID))
```

Comment:
@dorzepowski 
I thought it would be single endpoint `/transactions`
- and in this PR we "only" add additional filter parameter 🤔